source/_data/SymbioticLab.bib

```diff
@@ -1260,4 +1279,4 @@ @InProceedings{modelkeeper:nsdi23
     ModelKeeper achieves 1.3x-4.3x faster training completion
     with little overhead and no reduction in model accuracy.
   }
-}
+}
```

Comment:
```suggestion
}
```
Lines should end with a newline character.

Reply:
i cannot edit it to be this on my side?